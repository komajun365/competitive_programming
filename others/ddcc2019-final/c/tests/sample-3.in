0.00 0.00 0.90
0.65 0.65
3
