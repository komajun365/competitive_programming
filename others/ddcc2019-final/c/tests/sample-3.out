0.208804270061789
0.495936856319201
0.783069442576613
1
